### PR TITLE
Set log level in service runner

### PR DESCRIPTION
**Pull request by roaanv**
The log level set in the environment is never explicitly set.
This sets the log level as defined by the environment variable / settings

### CODE REVIEW

**Comment by roaanv** [comment]
@lyft/network-team

File: src/service_cmd/runner/runner.go [hunk 1]
```diff
@@ -9,18 +9,27 @@ import (
...
 )
 
 func Run() {
-	srv := server.NewServer("ratelimit", settings.GrpcUnaryInterceptor(nil))
-
 	s := settings.NewSettings()
 
+	logLevel, err := logger.ParseLevel(s.LogLevel)
+	if err != nil {
+		fmt.Printf("Could not parse log level. %v\n", err)
```
Comment (junr03) [inline src/service_cmd/runner/runner.go]: we probably want to exit if the intended log level can't be set. i.e `logger.Fatalf(<the message you have here>)`

File: src/service_cmd/runner/runner.go [hunk 2]
```diff
@@ -9,18 +9,27 @@ import (
 	pb "github.com/lyft/ratelimit/proto/envoy/service/ratelimit/v2"
 	pb_legacy "github.com/lyft/ratelimit/proto/ratelimit"
 
+	"fmt"
```
Comment (junr03) [inline src/service_cmd/runner/runner.go]: move this to the block with other standard library imports at the top.

**Comment by junr03** [comment]
We also need you to sign https://oss.lyft.com/cla/#!/signature

**Comment by junr03** [comment]
@roaanv We also need you to sign https://oss.lyft.com/cla/#!/signature

**Comment by roaanv** [comment]
I've signed the document. It doesn't seem to be picked up though.
image

**Comment by junr03** [comment]
@roaanv I think your github commits are not linked correctly to your github account (note the difference in the avatar of your commits) and the fact that it uses your name instead of your github username as the commiter name. Are your commits being signed by an email you have on github? You might want to rebase and squash all the commits into one in order to easily fix. 

Also, the commit still has the updated protos. This PR should only include the changes to the runner.go file.

**Comment by junr03** [comment]
@roaanv do you mind if I close this PR and put up a PR with the contents? I think it is pretty important to get this bug fix merged.

**Comment by roaanv** [comment]
@junr03 Sure, sorry I haven't had a chance to look at the issues.
Thanks for taking it up

**Comment by junr03** [comment]
No worries, thanks for bringing up!

**Pull request closed without merging**